ext/skeleton/skeleton.c

```diff
@@ -16,9 +16,9 @@
 	ZEND_PARSE_PARAMETERS_END()
 #endif
 
-/* {{{ void %EXTNAME%_test1()
```

Comment:
I am not sure if it's ok to get rid of the `%EXTNAME%` prefix, but it definitely makes it impossible to generate all the stubs